### PR TITLE
Hiding the mobile menu button on home page.

### PR DESCRIPTION
**Pull request by ymslavov**

### CODE REVIEW

File: resources/views/app.blade.php [hunk 1]
```diff
@@ -30,10 +30,12 @@
 				<img src="/assets/img/laravel-auditing-logo.png" alt="Laravel Auditing logo" style="margin-left: 15px;margin-right: 0;height: 40px;top: 14px;">
 				Laravel Auditing
 			</a>
-			
-			<div class="responsive-sidebar-nav">
-				<a href="#" class="toggle-slide menu-link btn">&#9776;</a>
-			</div>
+
+			@if (!Request::is('/'))
```
Comment (anteriovieira) [inline resources/views/app.blade.php]: This will hide the button in any media, is that right?

Comment (anteriovieira) [reply resources/views/app.blade.php]: I think you should use hidden-xs

File: resources/views/app.blade.php [hunk 2]
```diff
@@ -30,10 +30,12 @@
...
+				<div class="responsive-sidebar-nav">
```
Comment (anteriovieira) [inline resources/views/app.blade.php]: I think you should use the class hidden-xs

File: resources/views/app.blade.php (hunk 1 above)
Comment (ymslavov) [reply resources/views/app.blade.php]: Yes, any media.

File: resources/views/app.blade.php (hunk 2 above)
Comment (ymslavov) [reply resources/views/app.blade.php]: Yes, possibly the better solution, let me amend.

Comment (ymslavov) [reply resources/views/app.blade.php]: Actually, no, why would you want that button rendered on / at all? 

We'll need to do some ugliness like that 
```
<div class="responsive-sidebar-nav {{ Request::is('/') ? 'hidden-xs hidden-sm' : '' }}">
    <a href="#" class="toggle-slide menu-link btn">&#9776;</a>
</div>
```

Comment (anteriovieira) [reply resources/views/app.blade.php]: Sorry, you're right. It was my mistake.

**Comment by anteriovieira** [comment]
Perfect! Thanks @ymslavov 

www laravel-auditing com-

**Comment by ymslavov** [comment]
No worries!

**Pull request merged**